resources/assets/components/SignupCard/signup-card.scss

```diff
@@ -7,31 +7,6 @@
     color: $purple;
   }
 
-  .gallery {
-    @include clearfix;
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
-
-    > li {
-      margin: gutter() 0;
-      padding: 0 gutter();
-      overflow: hidden;
-
-      @include media($tablet) {
-        @include span(3 of 15);
-
-        &:first-child {
-          padding-left: 0px;
-        }
-
-        &:nth-of-type(5n + 1) {
-          clear: both;
-        }
-      }
-    }
-  }
-
```

Comment:
Just confirming  -- is this just not used any longer?

Reply:
From what I could tell, this is currently broken on `master` (showing zero posts for the first campaign, where I have 8) and all tiles are full-size:

![Screen Shot 2019-10-03 at 3 29 50 PM](https://user-images.githubusercontent.com/583202/66157909-c3b86a00-e5f2-11e9-9ebb-f01ee60ddf96.png)

 I just replaced with `.gallery.-quartet` which works out of the box:

![Screen Shot 2019-10-03 at 3 29 58 PM](https://user-images.githubusercontent.com/583202/66157913-c6b35a80-e5f2-11e9-8c76-5c35c7929b0a.png)

Reply:
YESSSSSS